src/foam/dao/daoUtils.js

```diff
@@ -284,7 +284,7 @@ foam.CLASS({
       isMerged: true,
       mergeDelay: 100,
       code: function() {
-        foam.localStorage.setItem(this.name, foam.json.stringify(this.array));
+        globalThis.localStorage.setItem(this.name, foam.json.stringify(this.array));
```

Comment:
Do we need the globalThis's everywhere? Since it's global, can't we just leave it out?

Reply:
Updated.